<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <widget class="QLineEdit" name="hostNameLineEdit">
      <property name="text">
       <string>192.168.1.100</string>
      </property>
     </widget>
    </item>
    <item row="1" column="11">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="2">
       <widget class="QLCDNumber" name="powerX2Lcd"/>
      </item>
      <item row="0" column="2">
       <widget class="QLCDNumber" name="powerX1Lcd"/>
      </item>
      <item row="1" column="1">
       <widget class="QLCDNumber" name="powerY1Lcd"/>
      </item>
      <item row="1" column="3">
       <widget class="QLCDNumber" name="powerY2Lcd"/>
      </item>
      <item row="1" column="2">
       <widget class="QLCDNumber" name="powerAllLcd"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="portLineEdit">
      <property name="text">
       <string>7777</string>
      </property>
     </widget>
    </item>
    <item row="3" column="10" colspan="2">
     <widget class="QPushButton" name="disconnectButton">
      <property name="text">
       <string>Disconnect</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QPushButton" name="connectButton">
      <property name="text">
       <string>Connect!</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QSlider" name="xSlider">
      <property name="minimum">
       <number>-200</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="12">
     <widget class="QwtPlot" name="plot" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QSlider" name="ySlider">
      <property name="minimum">
       <number>-200</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="logTextBrowser">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>465</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header location="global">qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
